src/text/text.rs

```diff
@@ -211,6 +211,8 @@ impl<'a> Text<'a> {
     /// Sets the alignment for this text.
     ///
     /// Defaults to: [`None`], meaning the alignment is determined by the rendering widget.
+    /// Setting the alignment of a Text generally overrides the alignment of its
+    /// parent Widget.
     ///
     /// Alignment can be set individually on each line to override this text's alignment.
```

Comment:
This line could also be reported on other shortcut methods.

Reply:
I also added this to the new convenience functions, or which functions do you mean?

Reply:
Aah I see, thanks for the clarification! Added it to the other methods